Assets/ScriptableObjects/Plugin Scripts/DisintegratePlugin.cs

```diff
@@ -13,10 +13,15 @@ public class DisintegratePlugin : BlockPlugin, IDisplaceable
     private const float Acceleration = 30f;
     private const float GravityAcceleration = 5f;
 
-    private void OnEnable()
+    private Animator _aminator;

```

Comment:
there's a typo here 😅

Reply:
This also means that animator is required in the main block for disintegrate plugin. I guess we can go with it for now but maybe in the future we can have animators for all blocks or some how do the animation within the plugin itself.

Reply:
good catch 🤣